### PR TITLE
move json files to root

### PR DESCRIPTION
**Pull request by flashdesignory**
Folder organization: this just moves all the json files into a dedicated `data` folder in the root of the repo.
Nothing else changed.

@felixarntz for awareness and feedback. Hopefully that'll make it easier for the php integration.

@kara

### CODE REVIEW

**Comment by felixarntz** [comment]
Thanks @flashdesignory! I opened #65 to update the PHP code accordingly. It would be great if you could merge that as soon as you've merged this one, or alternatively feel free to merge it into this PR so we can handle the changes altogether.

**Review by felixarntz — APPROVED** [review]

File: src/third-parties/google-tag-manager/index.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import data from './data.json';
+import data from '../../../data/google-tag-manager.json';
```
Comment (housseindjirdeh) [inline src/third-parties/google-tag-manager/index.ts:15]: Should we consider including absolute imports via alias (e.g. `@/data/...`)? Might be overkill and something we can worry about in a future PR

File: src/third-parties/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import GooglaAnalyticsData from './google-analytics/data.json';
-import GoogleTagManagerData from './google-tag-manager/data.json';
-import GoogleMapsEmbedData from './google-maps-embed/data.json';
-import GoogleYoutubeEmbedData from './youtube-embed/data.json';
+import GooglaAnalyticsData from '../../data/google-analytics.json';
```
Comment (housseindjirdeh) [inline src/third-parties/index.ts:1]: Seems like a typo from a previous PR squeezed through:

```suggestion
import GoogleAnalyticsData from '../../data/google-analytics.json';
```

**Review by housseindjirdeh — APPROVED** [review]
LGTM. Thanks for doing this 😍

**Pull request merged**